akka-stream/src/main/scala/akka/stream/scaladsl/Source.scala

```diff
@@ -98,6 +98,15 @@ final class Source[+Out, +Mat](
     (mat, Source.fromPublisher(pub))
   }
 
+  /**
+   * Connect this `Source` to the `Sink.ignore` and run it. Elements from the stream will be consumed and discarded.
+   *
+   * Note that the `ActorSystem` can be used as the implicit `materializer` parameter to use the
+   * [[akka.stream.SystemMaterializer]] for running the stream.
+   */
+  def run()(implicit materializer: Materializer): Future[Done] =
+    toMat(Sink.ignore)(Keep.right).run()
```

Comment:
Can be `.runWith` instead like the other `runNnn` methods.

Reply:
I don't understand what you exactly mean.

Reply:
Not important, but if you look at how for example `.runFold` is implemented it calls `runWith(sink)`. You can leave as is.